scalding-core/src/main/scala/com/twitter/scalding/bdd/TBddDsl.scala

```diff
@@ -68,7 +68,7 @@ trait TBddDsl extends FieldConversions with TypedPipeOperationsConversions {
 
       val outputPipe = operation(inputPipes) map { Tuple1(_) }
 
-      outputPipe.write(TypedTsv[Tuple1[OutputType]]("output"))
+      outputPipe.write(FixedPathTypedDelimited[Tuple1[OutputType]]("output", "\t"))
```

Comment:
do we need to make this change? Ideally, we don't need to make any change because we don't want to break a ton of code.